ush/set_rrfs_config_temp.sh

```diff
@@ -0,0 +1,297 @@
+
```

Comment:
Is it possible to put set_rrfs_config_temp.sh contents into set_rrfs_config.sh？

Reply:
I was thinking these two files should go to different places in config.sh. The file set_rrfs_config_temp.sh should be in the very beginning so we can load the common default settings depending on the grids and machine, and then make changes to overwrite them, if necessary. The file set_rrfs_config.sh goes to the end of config.sh, as it has some dependencies on what has been set in config.sh.
But if you have different ideas please feel free to make changes! Thanks!

Reply:
We need two set_rrfs files and one goes to the top and another goes to end of configure.